mmv1/templates/terraform/examples/kms_ekm_connection_basic.tf.erb

```diff
@@ -0,0 +1,12 @@
+resource "google_kms_ekm_connection" "<%= ctx[:primary_resource_id] %>" {
+  name            	= "<%= ctx[:vars]['ekmconnection_name'] %>"
+  location		= "us-central1"
+  key_management_mode 	= "MANUAL"
+  service_resolvers  	{
+      service_directory_service  = "projects/315636579862/secrets/external-servicedirectoryservice/versions/latest"
```

Comment:
This needs to be set up as a variable like in 
https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/cloudbuild_trigger_repo.tf.erb#L8
https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/cloudbuild/Trigger.yaml#L89

As it is now, this project will show in our documentation examples